Lib/webbrowser.py

```diff
@@ -610,10 +617,10 @@ def open(self, url, new=0, autoraise=True):
 
     # Don't clear _tryorder or _browsers since OS X can use above Unix support
     # (but we prefer using the OS X specific stuff)
-    register("safari", None, MacOSXOSAScript('safari'), -1)
-    register("firefox", None, MacOSXOSAScript('firefox'), -1)
-    register("chrome", None, MacOSXOSAScript('chrome'), -1)
-    register("MacOSX", None, MacOSXOSAScript('default'), -1)
+    register("safari", None, MacOSXOSAScript('safari'), True)
```

Comment:
With the above suggested change to the signature of `register`, this and the other options below would become:

    register("safari", None, MacOSXOSAScript('safari'), preferred=True)